code/src/components/Form.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import '../index.css'
+
+const Form = ({ messageNew, onMessageNewChange, onFormSubmit }) => {
+  return (
+    <>
+      <form onSubmit={onFormSubmit} className="form-container">
+        <label htmlFor="newMessage" className="main-title">
+          What's making you happy right now?
+        </label>
+
+        <textarea
+          id="newMessage"
+          type="text"
+          placeholder="React is making me happy!"
+          value={messageNew}
+          onChange={onMessageNewChange}
+          className="input-style"
+          rows="2"
+        />
+
+        <button type="submit" className="submit-button">
```

Comment:
An attribute like "max-length" could limit people from sending happy thoughts longer than 140 characters. I mean it's not like I want to limit people's happy thoughts, it's just that technigo told us to do that  🤷🏼‍♀️ 🙃  (I also refused to limit that.. I was like: send me those loooong happy thoughts!!! 😁)